mimic/rest/fastly_api.py

```diff
@@ -11,6 +11,18 @@
 from mimic.canned_responses import fastly
 
 
+def text_urldata(urldata):
+    """
+    Convert ``urldata`` into text.
+
+    :param urldata: a dictionary mapping bytes to lists of bytes.
+
+    :return: a dictionary mapping text to lists of text.
+    """
+    return dict([(k.decode("utf-8"), [vv.decode("utf-8") for vv in v])
+                 for [k, v] in urldata.items()])
```

Comment:
non-blocking: I think I saw this idiom somewhere else in the PR, should we consider refactoring it into a helper?